src/runtime_src/core/pcie/emulation/hw_em/generic_pcie_hal2/mbscheduler_hwemu.cxx

```diff
@@ -2248,6 +2309,7 @@ namespace hwemu {
         SCHED_DEBUGF("-> %s cmd(%lu)\n", __func__, xcmd->uid);
 
         xcmd->bo_init(buf);
+        convert_execbuf(xcmd);

```

Comment:
Looks like the return value is ignored?  This is why error handling with return values is bad, return codes are ignored by default.  It's better to use exceptions in c++ code, but I realize this is mostly copied from driver code where there are no exceptions so it's easier to stay close to that code.  That also goes for some of the other comments.